Makefile

```diff
@@ -11,7 +11,6 @@ lint:
 ## Build project
 build:
 	cargo build --verbose
-	cargo check
```

Comment:
[`cargo check` is just a syntax check; `cargo build` does `cargo check` + code gen so the call to `check` after is strictly unnecessary.](https://doc.rust-lang.org/cargo/commands/cargo-check.html)